app/views/admin/jobs/_form.html.erb

```diff
@@ -31,3 +36,13 @@
   <%= f.submit job.persisted? ? "Update" : "Create", class: "btn btn-primary" %>
   <%= link_to "View all jobs", admin_jobs_path, class: "btn btn-primary" %>
 <% end %>
+
+<script type="text/javascript">
```

Comment:
Wonder if this code ought to go in a js file somewhere as it's used a couple of times?